### DISCUSSION TITLE
Deb file hidden

### DISCUSSION THREAD

**Discussion by josephj11** [BurntSushi/ripgrep#2840] [General] [answered]
Came here to check it out. Almost gave up because I didn't want to mess with tarballs...

The deb file is there, (I'm on Kubuntu) but it was invisible until I clicked on display all assets.

Not sure what you could do to make it more visible, but you might get a few more users if it's easier to find.

**Comment by BurntSushi** [comment, marked as answer]
As you point out, I don't have any control over GitHub's UI. Thank you for the feedback though.

**Reply by BurntSushi** [reply]
Also, if you go to the release page itself, nothing is hidden: https://github.com/BurntSushi/ripgrep/releases/tag/14.1.0

I think things are only hidden when you're looking at a list of releases.
